lineapy/transformer/node_transformer.py

```diff
@@ -278,6 +279,30 @@ def visit_ImportFrom(self, node: ast.ImportFrom) -> None:
             attributes=create_lib_attributes(node.names),
         )
 
+    def visit_If(self, node: ast.If) -> Any:
+        test_call_node = self.visit(node.test)
+
+        if self.tracer.executor._id_to_value[test_call_node.id]:
+            exec = node.body
+            unexec = node.orelse
+        else:
+            exec = node.orelse
+            unexec = node.body
+        else_source = self.get_else_source(node)
+        self.tracer.control_node(
+            IfElseNode,
```

Comment:
change it to a enum listing all types of control nodes. This should be similar to the node_type parameter in the nodes.

Reply:
Using types.py::NodeType enum